trinity/tx_pool/pool.py

```diff
@@ -96,6 +105,8 @@ def _filter_tx_for_peer(
         return [
             val for val in txs
             if self._construct_bloom_entry(peer, val) not in self._bloom
+            # TODO: we need to keep track of invalid txs and eventually blacklist nodes
```

Comment:
Is there an issue open for this?  Did a quick search and didn't find one.  Seems like something that could be done as a "Good first issue" if you provide a sufficiently detailed spec for it.

Reply:
We have touched on it here. https://github.com/ethereum/py-evm/issues/893 Still need to settle on a design (but I was about to put out a draft in a PR soon)